Pod/ILGClasses/ILGClasses.m

```diff
@@ -66,9 +66,20 @@ + (NSSet *)subclassesOfClass:(Class)superclass
 
 + (NSSet *)classesConformingToProtocol:(Protocol *)protocol
 {
-    return [self classesPassingTest:^BOOL(Class class) {
+    NSSet *superclassesConforming = [self classesPassingTest:^BOOL(Class class) {
         return class_conformsToProtocol(class, protocol);
     }];
+    
+    NSMutableSet *subclasses = [NSMutableSet set];
+    
+    for (Class superclass in superclassesConforming) {
+        NSSet *subclassesOfSuper = [self subclassesOfClass:superclass];
+        [subclasses addObjectsFromArray:[subclassesOfSuper allObjects]];
+    }
+    
+    NSSet *allConforming = [superclassesConforming setByAddingObjectsFromSet:subclasses];
+    
+    return allConforming;
```

Comment:
Is all this necessary or would changing `class_conformsToProtocol(class, protocol)` to `[class conformsToProtocol:protocol]` on line 70 do the trick?